platform/database_objects/config_cloud.py

```diff
@@ -0,0 +1,10 @@
+# string to configure the database connection - localhost (will have to change to cloud in the future)
+# Database parameters for local MySQL database
+local_host = "localhost"         
```

Comment:
i think this should be an environment variable. Locally, when initing database, we should have a script to set these variables. 

in github, we can set these environment variables securely to connect them to live db  

@cherrythia @tkpython what do yall think?

Reply:
yeah, i agree. Need to be inside an environment file, .env.local and .env.production to switch between local and prod database. Because the Prod database secret password, by right i need to encrypt it using github secret before storing into github. 

Is this commonly used in python? 
https://pypi.org/project/python-dotenv/